cfpb/cfgov-refresh
# Update watch and clean front-end gulp tasks

JS and Less files were being ignored for projects that live in `/unprocessed/apps/`.

## Changes

- Gulp's watch task now watches `paths.unprocessed + '/apps/**/[js|css]/**/*.[js|css]'`
- Gulp's clean task now cleans `paths.unprocessed + '/apps/**/[js|css]/**/*.[js|css]'`

## Testing

Clean task

1. `gulp` to build everything.
1. `/cf_gov/static_built/apps/regulations3k/css/` and `/cf_gov/static_built/apps/regulations3k/js/` should contain some files.
1. `gulp clean:css` and `gulp clean:js` should delete those files.

Watch task

1. `gulp watch`
1. Edit `/cfgov/unprocessed/apps/regulations3k/js/module1.js` and see if it recompiles the JS.
1. Edit `/cfgov/unprocessed/apps/regulations3k/css/layout.less` and see if it recompiles the CSS.
